### PR TITLE
changed DI syntax to work with minification

### PR DESCRIPTION
**Pull request by civilframe**
@nnnnathann ,

I'm not sure why grunt generated a lot of other changes on top of mine. I assume this is part of your build process. I will comment on the lines that I have specifically changed.

Thanks!

### CODE REVIEW

File: dist/angular-panhandler.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
-/*! angular-panhandler - v1.0.2 - 2015-02-15
+/*! angular-panhandler - v1.0.3 - 2015-07-09
 * Copyright (c) 2015 ; Licensed MIT %> */
 (function(){
   'use strict';
   angular.module('panhandler', [])
-    .directive('panhandler', function PanhandlerFactory($document) {
-      PanhandlerFactory.$inject = ['$document'];
+    .directive('panhandler', ['$document', function PanhandlerFactory($document) {
```
Comment (civilframe) [inline dist/angular-panhandler.js]: This is the only material change.

**Comment by civilframe** [comment]
PS. I am using this in my project https://github.com/AppFactoryOrg/app-factory Works great :+1:

**Comment by nnnnathann** [comment]
Apologies for the late response, didn't see this until now.

Merged in v1.1.2, thank you for the effort, and for using the library! :)

**Comment by civilframe** [comment]
Glad I could help. This library is super useful for my project https://github.com/AppFactoryOrg/app-factory 😁

**Pull request closed without merging**